dothething/dothething/Views/ProfileView.swift

```diff
@@ -264,7 +264,7 @@ extension ProfileView {
                             // save url to clips array
                             DispatchQueue.main.async {
                                 let thumbnail = Thinger.getThumbnail(url: tempFileUrl, degreesToRotate: self.videoDegreesToRotate)
-                                self.clips.append(Clip(url: tempFileUrl, thumbnail: thumbnail, isHighlighted: false, metadata: clip))
+                                self.clips.append(Clip(url: tempFileUrl, thumbnail: thumbnail, isHighlighted: false, metadata: clip, showCode: true))
```

Comment:
The change to the Clip initialization in the addClipFromUrl function is correct, as the showCode property has been added to the Clip instance being created.